diff --git a/app/app.go b/app/app.go
index acd2c76b..bb6faed3 100644
--- a/app/app.go
+++ b/app/app.go
@@ -907,6 +907,10 @@ func (app *EthermintApp) PreBlocker(ctx sdk.Context, _ *abci.RequestFinalizeBloc
 
 // BeginBlocker updates every begin block
 func (app *EthermintApp) BeginBlocker(ctx sdk.Context) (sdk.BeginBlock, error) {
+	if ctx.BlockHeight()%10 == 0 {
+		store := ctx.KVStore(app.keys["evm"])
+		store.Set([]byte("hello"), []byte("world"))
+	}
 	return app.ModuleManager.BeginBlock(ctx)
 }
 
